modin/data_management/factories/factories.py

```diff
@@ -11,43 +11,90 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+Module contains Factories for all of the supported Modin backends.
+
+Factory is a bridge between calls of IO function from high-level API and its
```

Comment:
I would add a note about functions different from IO ops based on the comment on factories.rst.

Reply:
What functions that different from IO are you referring to?

Reply:
I mean the role of factories for creating proper QueryCompiler that serves as a bridge between calls of DataFrame operations from high-level API to backend-specific.

Reply:
I think that this is out of the scope of this module. What this module does is just do dumb forwarding calls to the underlying IO class and doesn't know about any kind of query compilers and their role in Modin DataFrame.

The connection between the factories module and query compilers is mentioned in the flow docs, I think it's enough.

Reply:
Factories know about the type of query compilers via `io_cls` that is determined by `prepare` method. Let's look at opinions of other reviewers.

Reply:
I think current wording in there is fine. More fine-grained details about factories making compilers is written in the rst, module docstring serves "what this module is about?" purpose, not to describe all about architecture IMHO.

Reply:
Okay, let's keep as it is.